packages/gatsby-theme-newrelic/src/components/Link.js

```diff
@@ -13,98 +13,86 @@ const isExternal = (to) => to.startsWith('http');
 const isNewRelic = (to) => to.startsWith('https://newrelic.com');
 const isSignup = (to) => to.startsWith('https://newrelic.com/signup');
 
-// https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a
-const ATTRIBUTES = [
```

Comment:
This is missing all of the event handlers for the `Link`, which caused the hover styling to break on the new search experience. Because there are lots of different handlers that can be used for links, I'd rather find and fix the areas of the apps that send invalid props to the `Link` component. Thats going to be easier than trying to maintain a whitelist of "valid" props here, since we don't have to track all of the different handlers that can be passed to these components (e.g. `onMouseOver`, `onMouseOut`, etc.).